persistence/upgrades/ravendb-5to6.md

```diff
@@ -165,6 +165,61 @@ In NServiceBus versions 3 and 4, RavenDB Persistence was part of the main NServi
 
 If attempting to convert directly from NServiceBus 4 or lower to NServiceBus 7 (which is _not recommended_) timeout data will have to be converted to the new format manually.
```

Comment:
@danielmarbach I think this statement is incorrect. It doesn't matter is you migrated one version at a time or all at once. RavenDB will not migrated IDLE data AFAIK.

Reply:
When I did the research for this we saw that an entity conversion listener was available in previous versions that converted timeouts due into the new format. So why are you know saying this is not correct?

Reply:
@danielmarbach The converter is AFAIK only triggered once the timeout expires when read from storage. Meaning, any timeouts that did not yet expire prior to 6 but will after migrating to v6 cannot be converted.

Reply:
Exactly and that is what this wording says. Because as long as you run the previous versions no manual conversion is necessary. New timeouts are already stored in the new format and old timeouts are converted upon deserialization like the previous paragraph mentions. I think the text assumes there are no timeouts left anymore because you went through multiple versions of core. 

Maybe the wording is just a bit confusing and should be changed to something like

If timeouts generated by the persistence targeting NServiceBus 4 are known to be due timeout data will have to be converted to the new format manually.

What do you think?

Reply:
@danielmarbach I'll create a new PR and add you as a reviewer as I see that you are on leave.